src/components/VCheckbox/VCheckbox.vue

```diff
@@ -7,26 +7,44 @@
       v-bind="inputAttrs"
       @change="onChange"
     />
-    <Checkmark
+    <svg
       v-show="localCheckedState"
-      class="checkmark"
+      class="checkmark stroke-current"
       focusable="false"
+      viewBox="0 0 24 24"
       width="20"
       height="20"
       aria-hidden="true"
-    />
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <use :href="`${checkmark}#icon`" />
+    </svg>
```

Comment:
I didn't realize we can use `VIcon` here because the size was different, and because it was an absolutely positioned element, but we actually can, and it makes the component better!